### ISSUE TITLE
Decreased Dehyroepiandrosterone serum concentration

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6885]
**Preferred term label:**
Decreased dehyroepiandrosterone serum concentration

**Definition (free text, please give PubMed ID)**
Decrease of dehyroepiandrosterone level  in serum 
20301592[PMID]; 3312397[PMID]

**Parent term (use hpo.jax.org/app)**
Abnormal circulating androgen level (HP:0030347)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital adrenal hyperplasia due to cytochrome P450 oxidoreductase deficiency (ORPHA 95699)

**Comment by pnrobinson** [comment]
This term existed already, https://hpo.jax.org/app/browse/term/HP:0031214
